cvat/apps/engine/static/engine/js/annotationParser.js

```diff
@@ -16,7 +16,6 @@ class AnnotationParser {
         this._parser = new DOMParser();
         this._startFrame = job.start;
         this._stopFrame = job.stop;
-        this._flipped = job.flipped;
```

Comment:
It is OK to delete it from UI.

Reply:
We can't remove only server part or only client part. Anyway something will be wrong. I describe above how we can solve the problem.

Reply:
We can hide it from UI for all new tasks but I agree that a migration file is a better solution.